web/thesauruses/vbnet/16/operators.json

```diff
@@ -0,0 +1,200 @@
+{
+  "meta": {
+    "language": "vbnet",
+    "language_version": "16.0",
+    "language_name": "Visual Basic",
+    "structure": "operators"
+  },
+  "concepts": {
+    "addition": {
+      "code": "+",
```

Comment:
Could you add where operands would go on this? Like:
`"code": "operand1 + operand2",`

Some languages shift where they're at and this would be helpful.

Reply:
Yes. So I am assuming you want me to add this for all operators not just addition?

Reply:
Yes. All operators. Thanks!

Reply:
@geekygirlsarah I have added the operands